index.d.ts

```diff
@@ -2,6 +2,7 @@ export const originalConsole = Console;
 
 type Options = {
   isSilent?: boolean;
+  stripsAnsi?: boolean;
```

Comment:
I'd prefer just `stripAnsi`, seems unnecessary to add the `s` here.

Reply:
@kevin940726 Fixed